atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -315,66 +473,33 @@ def get_build(self, metadata, worker_metadatas):
-        extra['image']['isolated'] = isolated
-
-        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
-        if fs_result is not None:
-            try:
-                fs_task_id = fs_result['filesystem-koji-task-id']
-            except KeyError:
-                self.log.error("%s: expected filesystem-koji-task-id in result",
-                               AddFilesystemPlugin.key)
-            else:
-                try:
-                    task_id = int(fs_task_id)
-                except ValueError:
-                    self.log.error("invalid task ID %r", fs_task_id, exc_info=1)
-                else:
-                    extra['filesystem_koji_task_id'] = task_id
-
-        extra['image'].update(get_parent_image_koji_data(self.workflow))
-
-        flatpak_source_info = get_flatpak_source_info(self.workflow)
-        if flatpak_source_info is not None:
-            compose_info = get_compose_info(self.workflow)
-            koji_metadata = compose_info.koji_metadata()
-            koji_metadata['flatpak'] = True
-            extra['image'].update(koji_metadata)
-
         koji_task_owner = get_koji_task_owner(self.session, koji_task_id).get('name')
         extra['submitter'] = self.session.getLoggedInUser()['name']
 
-        resolve_comp_result = self.workflow.prebuild_results.get(PLUGIN_RESOLVE_COMPOSES_KEY)
-        if resolve_comp_result:
-            extra['image']['odcs'] = {
-                'compose_ids': [item['id'] for item in resolve_comp_result['composes']],
-                'signing_intent': resolve_comp_result['signing_intent'],
-                'signing_intent_overridden': resolve_comp_result['signing_intent_overridden'],
-            }
-        if self.workflow.all_yum_repourls:
-            extra['image']['yum_repourls'] = self.workflow.all_yum_repourls
-
-        self.set_help(extra, worker_metadatas)
         self.set_media_types(extra, worker_metadatas)
-        self.set_go_metadata(extra)
-        self.set_operators_metadata(extra, worker_metadatas)
-        self.set_group_manifest_info(extra, worker_metadatas)
-
-        build = {
-            'name': component,
-            'version': version,
-            'release': release,
-            'source': "{0}#{1}".format(source.uri, source.commit_id),
-            'start_time': start_time,
-            'end_time': int(time.time()),
-            'extra': extra,
-            'owner': koji_task_owner,
-        }
+
+        if self.source_build:
+            build = {
+                'name': self.workflow.koji_source_nvr['name'],
+                'version': self.workflow.koji_source_nvr['version'],
+                'release': self.workflow.koji_source_nvr['release'],
+                'source': self.workflow.koji_source_source_url,
+                'start_time': start_time,
+                'end_time': int(time.time()),
+                'extra': extra,
+                'owner': koji_task_owner,
```

Comment:
Could set the common values first, then update with the values that vary depending on build type.

Reply:
sure